app/controllers/api/v1/authentication_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Api::V1::AuthenticationController < ApplicationController
+  class AuthenticationError < StandardError; end
+  include ActionController::HttpAuthentication::Token
+
+  rescue_from ActionController::ParameterMissing, with: :parameter_missing
+  rescue_from AuthenticationError, with: :handle_unauthenticated
+
+  def create
+    user = User.find_by(email: params.require(:email))
+    raise AuthenticationError unless user.authenticate(params.require(:password))
+
+    token = AuthenticationTokenService.call(user.id)
+
+    render json: { token: }, status: :created
+  end
+
+  def token_validation
+    # Authorization: Bearer <token>
+    token, _options = token_and_options(request)
```

Comment:
I get an error from the authentication service which seems to be coming from this line: 
- "#<JWT::DecodeError: Nil JSON web token>"

![image](https://user-images.githubusercontent.com/44676055/179966775-67d52bb7-129a-4a2c-b4f7-468bbdccc042.png)


The token in line 19 gives nill. I think the previous function: 

token = request.headers['Authorization'] 
was a better way of picking the token as I see you solved the previous error in the Auth service